server/protocols/webadmin/webadmin-data/src/test/java/org/apache/james/webadmin/routes/AliasRoutesTest.java

```diff
@@ -72,14 +72,11 @@
     public static final String BOB_ALIAS_WITH_SLASH = "bob-alias/@" + DOMAIN.name();
     public static final String BOB_ALIAS_WITH_ENCODED_SLASH = "bob-alias%2F@" + DOMAIN.name();
     public static final String ALICE = "alice@" + DOMAIN.name();
+    public static final String ALICE_ALIAS = "alice-alias@" + DOMAIN.name();
     public static final String BOB_PASSWORD = "123456";
     public static final String BOB_WITH_SLASH_PASSWORD = "abcdef";
     public static final String ALICE_PASSWORD = "789123";
 
-    private static final MappingSource BOB_SOURCE = MappingSource.fromUser("bob", DOMAIN);
-    private static final MappingSource BOB_WITH_ENCODED_SLASH_SOURCE = MappingSource.fromUser("bob/", DOMAIN);
-    private static final Mapping BOB_MAPPING = Mapping.alias(BOB_ALIAS);
```

Comment:
Héhé that was the other way around: When I send a mail to the alias then bob receives it!

We should have :

```
    private void addAlias(MappingSource source, MailAddress aliasSourceAddress) throws RecipientRewriteTableException {
        try {
            recipientRewriteTable.addAliasMapping(MappingSource.fromMailAddress(alias), bobAddress.asString());
        } catch (MappingAlreadyExistsException e) {
            // ignore
        }
    }
```

Edit! Fix the previous code in a separate, earlier commit!